src/rust/engine/process_execution/src/local.rs

```diff
@@ -90,6 +103,7 @@ impl super::CommandRunner for CommandRunner {
     let env = req.env;
     let output_file_paths = req.output_files;
     let output_dir_paths = req.output_directories;
+    let cleanup_local_dirs = self.cleanup_local_dirs.clone();
```

Comment:
`.clone()` on a type which implements `Copy`, like `bool`, is unnecessary - the compiler will automatically insert copies where it needs to. Just `let cleanup_local_dirs = self.cleanup_local_dirs;` should be fine here.

Reply:
done